tokio/build.rs

```diff
@@ -19,12 +19,13 @@ const CONST_MUTEX_NEW_PROBE: &str = r#"
 
 const AS_FD_PROBE: &str = r#"
 {
-    #![allow(unused_imports)]
-
+    #[allow(unused_imports)]
     #[cfg(unix)]
     use std::os::unix::prelude::AsFd as _;
+    #[allow(unused_imports)]
     #[cfg(windows)]
     use std::os::windows::prelude::AsSocket as _;
+    #[allow(unused_imports)]
     #[cfg(target = "wasm32-wasi")]
```

Comment:
Btw, I don't think `#[cfg(target = "..")]` is a valid cfg. (we should use `#[cfg(target_os = "wasi")]`)